corehq/ex-submodules/couchexport/models.py

```diff
@@ -110,820 +71,5 @@ def from_format(cls, format):
-
-    @property
-    def type(self):
-        # hack - make this backwards compatible with form/case categorization
-        # these might only exist in the care-bihar domain or wherever else
-        # they've been manually created in the DB.
-        try:
-            return 'form' if 'http:' in self.index[1] else 'case'
-        except IndexError:
-            # arbitrarily choose default so it doesn't stay hidden from the UI forever.
-            return 'form'
-
-    def __repr__(self):
-        return '%s (%s)' % (self.name, self.index)
-
-
-class GroupExportComponent(object):
-    """
-    Helper wrapper class for components of a GroupExportConfiguration
-    """
-
-    def __init__(self, config, saved_version, group_id, index):
-        self.config = config
-        self.saved_version = saved_version
-        self.group_id = group_id
-        self.index = index
-
-
-class GroupExportConfiguration(Document):
-    """
-    An export configuration allows you to setup a collection of exports
-    that all run together. Used by the management command or a scheduled
-    job to run a bunch of exports on a schedule.
-    """
-    full_exports = SchemaListProperty(ExportConfiguration)
-    custom_export_ids = StringListProperty()
-
-    def get_custom_exports(self):
-        for custom in list(self.custom_export_ids):
-            custom_export = self._get_custom(custom)
-            if custom_export:
-                yield custom_export
-
-    def _get_custom(self, custom_id):
-        """
-        Get a custom export, or delete it's reference if not found
-        """
-        try:
-            return SavedExportSchema.get(custom_id)
-        except ResourceNotFound:
-            try:
-                self.custom_export_ids.remove(custom_id)
-                self.save()
-            except ValueError:
-                pass
-
-    @property
-    @memoized
-    def all_configs(self):
-        """
-        Return an iterator of config-like objects that include the
-        main configs + the custom export configs.
-        """
-        return [full for full in self.full_exports] + \
-               [custom.to_export_config() for custom in self.get_custom_exports()]
-
-    @property
-    def all_export_schemas(self):
-        """
-        Return an iterator of ExportSchema-like objects that include the
-        main configs + the custom export configs.
-        """
-        for full in self.full_exports:
-            yield DefaultExportSchema(index=full.index, type=full.type)
-        for custom in self.get_custom_exports():
-            yield custom
-
-    @property
-    @memoized
-    def all_exports(self):
-        """
-        Returns an iterator of tuples consisting of the export config
-        and an ExportSchema-like document that can be used to get at
-        the data.
-        """
-        return list(zip(self.all_configs, self.all_export_schemas))
-
-
 class SavedBasicExport(BlobMixin, Document):
```

Comment:
Is this still used anywhere except blob db tests? If not, those tests can be removed. I'll be happy to do that in a separate PR if that would be easier.

Reply:
It's used for metadata in something related to blobdb or blob meta. I figured I didn't understand that system well enough to know whether removing one of the options would cause an issue, and it was somewhat tangential to the overall thing I was trying to do, so i didn't try to hard to reason with it.

Reply:
Removed in 84dacb19bfcfc4385a00ea8d10a13a63fdedc53a